src/main/java/com/datasonnet/header/Header.java

```diff
@@ -127,13 +129,25 @@ public static Header parseHeader(String script) throws HeaderParseException {
 
         String version = versionMatcher.group("version");
         String headerWithoutVersion = headerSection.substring(versionMatcher.end());
+        String[] splitVersion = version.split("\\.",2); //[0] = major [1] = minor + remainder if exists
 
-        if(VERSION_1_0.equals(version)) {
-            return parseHeader10(headerWithoutVersion);
-        } else if(VERSION_2_0.equals(version)) {
-            return parseHeader20(headerWithoutVersion);
-        } else {
-            throw new HeaderParseException("Version must be one of 1.0 or 2.0 but is " + version);
+        switch (splitVersion[0]) {
+            case "1":
+                if ("0".equals(splitVersion[1])) {
+                    return parseHeader10(headerWithoutVersion);
+                }
+                throw new HeaderParseException("Version must be 1.0 but is " + version);
+            case "2":
+                if ("0".equals(splitVersion[1])) {
+                    return parseHeader20(headerWithoutVersion);
+                } else {
+                    //not sure if a print out is a good enough warning or if we want to add loggers
+                    System.out.println("WARNING: You are using a version that is still in development. " +
```

Comment:
Since we haven't set up loggers yet I think printing is fine(for now), but it should be a print to System.err